testing/tests/moduletests/LCR/LCR_02.cpp

```diff
@@ -83,10 +76,7 @@ namespace test
     TEST(LCR_02, T04)
     {
         SETUP()
-        StateGui tele;
-        Mem::zero(tele);
-        tele.state1 = 401;
-        tele.state2 = 402;
+        StateGui tele(401, 402);
         
         STEP(1)
         m_Dispatcher().expectDispatch(4, tele);
```

Comment:
would it be possible to make expectations by rvalue?
please check!

Reply:
yes, implemented